Modules/expat/expat.h

```diff
@@ -342,7 +342,7 @@ XML_SetEntityDeclHandler(XML_Parser parser,
                          XML_EntityDeclHandler handler);
 
 /* OBSOLETE -- OBSOLETE -- OBSOLETE
-   This handler has been superseded by the EntityDeclHandler above.
+   This handler has been superceded by the EntityDeclHandler above.
```

Comment:
> it is... but lets not correct spelling in third party code.

Right, I prefer to reduce the downstream changes: so I reverted this change as well.